src/main/cpp/subsystems/Winch.cpp

```diff
@@ -7,12 +7,30 @@
 
 #include "subsystems/Winch.h"
 
+constexpr double kMaxVelocityError{4000-3000};
```

Comment:
These numbers seem like they were copied from DooHickey. We need to do some measurements for the Winch.